packages/main/src/plugin/extension-loader.ts

```diff
@@ -1036,7 +1075,7 @@ export class ExtensionLoader {
     }
   }
 
-  async deactivateExtension(extensionId: string): Promise<void> {
+  async deactivateExtension(extensionId: string, system?: boolean): Promise<void> {
```

Comment:
quickly looking at the `system` parameter I don't understand what it will do. Like is it because it's a system extension ?

maybe rename it to `shutdown` or `systemShutdown` or `systemOrder` ?

or just what is triggering the call: `stopAllExtensions`